site/en/blog/chips-origin-trial-extended/index.md

```diff
@@ -17,26 +18,32 @@ tags:
   - cookies
 ---
 
-[CHIPS](/docs/privacy-sandbox/chips/) is a Privacy
-Sandbox proposal that introduces a mechanism to opt-in to having third-party
-cookies partitioned by top-level site using a new cookie attribute,
-`Partitioned`.
+[CHIPS](/docs/privacy-sandbox/chips/) is a Privacy Sandbox proposal that
+introduces a mechanism to opt-in to having third-party cookies partitioned by
+top-level site using a new cookie attribute, `Partitioned`.
+
+{% Aside %}
+
+**Update:** To give developers more time to test the feature and gather more
+feedback, an [extension to the experiment will run until October 25, 2022 around
+the end of the Chrome 106 Stable
+release](https://groups.google.com/a/chromium.org/g/blink-dev/c/MKQODOL0Fso/m/S3ss7A8jCAAJ)
+.
+
+{% endAside %}
```

Comment:
Same comment as above on the "end of the release" bit.

Reply:
Ditto.